components/footer.js

```diff
@@ -1,24 +1,34 @@
+import Link from 'next/link'
+
 const footer = () => (
     <footer className="footer">
         <div className="columns">
             <div className="column">
                 <div className="title is-4 red has-text-centered">
-                    <h1>Join</h1>
+                    <Link href="/join">
+                        <h1>Join</h1>
```

Comment:
Replace the h1 with an `a` tag with a className of title. EX:
`<h1>Join</h1>` should become `<a className="title">Join</a>`